plat/mediatek/mt6795/bl31.ld.S

```diff
@@ -0,0 +1,198 @@
+/*
```

Comment:
I've added an issue for this: https://github.com/ARM-software/tf-issues/issues/364
But we haven't committed to doing this yet.